spec/ParseGraphQLServer.spec.js

```diff
@@ -476,7 +498,7 @@ describe('ParseGraphQLServer', () => {
               const {
                 response: { headers },
               } = context;
-              expect(headers.get('access-control-allow-origin')).toEqual('*');
+              expect(headers.get('access-control-allow-origin')).toEqual('http://someorigin.com');
```

Comment:
Is it a breaking change?

Reply:
No this is a security enhancement, also discussed with The Guild maintainers.
The server still accepts `*` but the response auto-detect the incoming origin and  "scope" the response to this origin. It's an improved security mechanism included out of the box in Yoga.

Reply:
Please use ' example.com` to not violate RFC.